src/OptionSettlement.sol

```diff
@@ -70,7 +71,7 @@ contract OptionSettlementEngine is ERC1155 {
     address public feeTo = 0x36273803306a3C22bc848f8Db761e974697ece0d;
 
     // To increment the next available token id
-    uint256 private _nextTokenId;
+    uint256 public _nextTokenId;
```

Comment:
If we want this to be public then we should remove the leading underscore. I'm not sure about making it public.

Reply:
as mentioned above, changed _nextTokenId and chainMap to public just for testing. maybe we can change it to private at the end?

Reply:
Ok, then let's just remove the leading underscore here, which denotes a private storage variable.

Reply:
done.